### DISCUSSION TITLE
Persist `podman` installation after firmware update

### DISCUSSION THREAD

**Discussion by amomchilov** [unifi-utilities/unifi-common#363] [Q&A]
I had to re-install `podman` on my UDM Pro SE this morning after a firmware update.

Is there a way to make it persistent?

**Comment by an9rch1st** [comment]
podman, pi-hole and mcast relay containers were lost after upgrade yesterday.

**Comment by twojenski** [comment]
Same problem with my UDM Pro SE as well.
Latest update wiped out everything.

Was able put podman back on and run zerotier, without any problems.
When I tried to install pihole, it podman runs it, but I can not connect via HTTP to 10.0.5.3;  ping 10.0.5.3 works.

Anybody have problems getting pihole to run again?

**Reply by amomchilov** [reply]
> Anybody have problems getting pihole to run again?

I basically had to go through the installation steps all over again.

**Reply by twojenski** [reply]
Yea...  not good.

**Comment by GigaBypass9876** [comment]
I had the same experience, reinstalling podman, removing dns and ip link, then running
/mnt/data/on_boot.d/10-dns.sh
/mnt/data/on_boot.d/05-install-cni-plugins.sh
/mnt/data/on_boot.d/custom_pihole_dote.sh 
got it back up. 

Has anyone with a UDMP-SE had any luck with utilities functioning after reboot using symlinks as suggested in #321 here?
https://github.com/unifi-utilities/unifios-utilities/issues/321#issuecomment-1034079344
